data/videos/deadheadanimation/JYpbrsc_fpg.yml

```diff
@@ -1,6 +1,6 @@
 id: JYpbrsc_fpg
 title: "𝐈𝐧𝐟𝐚𝐧𝐭𝐢𝐥𝐢𝐳𝐢𝐧𝐠/𝐥𝐞𝐠𝐢𝐭𝐢𝐦𝐢𝐳𝐢𝐧𝐠 𝙷𝚘𝚠 𝚝𝚘: 𝚁𝚎𝚟𝚘𝚕𝚞𝚝𝚒𝚘𝚗 ᵖᵃʳᵗ⁴"
```

Comment:
Concerned about the accessibility aspect of these "special characters". I would be keen to check how a screen reader handles them when I get a chance!

Reply:
Until I've had a chance to do that I am going to make as blocked.